example/styles.ts

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from 'react';
+
+import styled from 'react-native-styled.macro';
+import { useWindowVariant } from 'react-native-styled.macro/lib';
+
+export function useStyles() {
+	const variants = useWindowVariant();
+
+	const styles = useMemo(() => {
+		return {
+			button: {
+				...styled(
+					[
+						'mx-2',
+						'my-4',
+						'border',
+						'rounded-full',
+						'border-teal-600',
+						'bg-teal-600',
+						'sm:bg-green-600',
+						'md:bg-teal-600',
+						'lg:bg-yellow-600',
+						'xl:bg-orange-600',
+					],
+					variants
+				),
+			},
+			buttonText: {
+				...styled(
+					['px-2', 'py-1', 'text-base', 'text-white'],
+					variants
+				),
+			},
+			container: {
+				...styled(['flex-1', 'items-center'], variants),
```

Comment:
These two components (`buttonText`, `container`) don't need `variants` as their styles will never change at runtime.

Reply:
Ah! I see.  Copy pasta. I'll update it when I submit a change to all this.

Reply:
The macro is smart enough to detect that and automatically optimize the code btw but still removing these will make the code a bit cleaner :)